### ISSUE TITLE
Pipeline Only Mode

### ISSUE THREAD

**Issue by Lite5h4dow** [harness/gitness#3518]
Given that Gitness builds on Drone, are there plans to allow someone deploying this to disable the git repo management side of this application?

I tried Gitness for a good few months. but with it missing issues, projects, discussions, time tracking and all the other features my workflow requires, I decided to switch to Gitea and use Drone instead. now that I've been building out my repos and such in Gitea, I don't want to switch again even if gitness does end up getting all the features, I don't want to be in a situation where I'm forced to either switch to Gitness down the line or be stuck with Drone as it slowly decays from Harness' priority list.

Basically, I'm asking if I should hold onto Drone until gitness launches, fully featured and replace Drone or if I should be looking to replace Drone now with some other CI/CD Pipeline system.

**Comment by bradrydzewski** [comment]
hey there, please see this thread which should answer you question.  The tldr; is that we are working on supporting pipelines for third party repositories and we will never force anyone to host their code in Gitness
https://github.com/harness/gitness/issues/3374#issuecomment-1775189935

**Comment by Lite5h4dow** [comment]
thanks for coming back not only so quick but with a definitive answer, would be nice if this was on the website somewhere (unless it is and I missed it in my diving) also 

> I'm hoping this capability will land in the next 3-4 weeks.

yeah.... about that chief....
